ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PauseResumeIntegrationTest.java

```diff
@@ -125,6 +127,13 @@ public void shouldPauseAndResumeQuery() {
     RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "PAUSE " + queryId + ";");
 
     assertThat(getPausedCount(), equalTo(1L));
+    REST_APP.getEngine().metricCollectors().getMetrics();
```

Comment:
Is this call necessary? The return value is ignored.

Reply:
Nope, it is going.